orttraining/orttraining/models/bert/main.cc

```diff
@@ -367,6 +361,15 @@ Status ParseArguments(int argc, char* argv[], BertParameters& params, OrtParamet
     ORT_RETURN_IF_NOT(params.data_parallel_size > 0, "data_parallel_size must > 0");
     ORT_RETURN_IF_NOT(params.horizontal_parallel_size > 0, "horizontal_parallel_size must > 0");
 
+    int seed = flags["seed"].as<int>();
+    if (params.horizontal_parallel_size > 1 && seed <= 0) {
+      seed = 8211; // Megatron needs a random seed.
+    }
+    if (seed > 0) {
+      utils::SetRandomSeed(seed);
+      printf("Random seed is set to %d.\n", seed);
```

Comment:
NIT: Should we use std::cout instead of printf? I noticed that tehre are existing printf, but I am not sure whether there is a guideline where we should upgrade new logs to std:;cout instead